### PR TITLE
#159284989 Fixed UI fonts, menu and width of entry title input  

### PR DESCRIPTION
**Pull request by EphraimDev**
### What does this PR do?
Implements feedback on my UI templates
### Description of Task to be completed?
Changes the font, MyDiary menu position on header and width of title box on add entry and edit entry page
#### How should this be manually tested?
After cloning the repo, pages can be viewed locally
#### What are the relevant pivotal tracker stories?
#159284989
#### Questions:
I can't seem to merge this branch with my gh-pages branch, Do I commit to the branch separately so my changes can be hosted and viewed appropriately?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+fabric8-analytics.log
+.nyc_output/
+coverage/
+package-lock.json
+.coveralls.yml
+alternative/
+alternative-tmp/
+out/
+.env
+build/
```
Comment (louisdante9) [inline .gitignore:11]: pls add vsccode to the list

File: UI/addEntry.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```
Comment (louisdante9) [inline UI/addEntry.html]: html file name shouldn't use camel casing

File: UI/css/addentry.css [hunk 3]
```diff
@@ -0,0 +1,2300 @@
+body {
```
Comment (louisdante9) [inline UI/css/addentry.css]: pls avoid using camel casing for file names

Comment (louisdante9) [reply UI/css/addentry.css]: how is it that you have 2207 lines of css written within a day?

**Review by louisdante9 — COMMENTED** [review]
Good job working on your feedback but i would like you to give me some info on why you have 56 file changes and most of your css file changes are much. they go over 2000 lines of code

File: .gitignore (hunk 1 above)
Comment (EphraimDev) [reply .gitignore:11]: Done, thanks

File: UI/addEntry.html (hunk 2 above)
Comment (EphraimDev) [reply UI/addEntry.html]: Will change them right away

File: UI/css/addentry.css (hunk 3 above)
Comment (EphraimDev) [reply UI/css/addentry.css]: I have so many media queries for the different screen sizes. What I did was write the styling for one screen size, copy the styling and add it to another screen size, and if adjustments have to be made, I effect same.

File: UI/addEntry.html (hunk 2 above)
Comment (EphraimDev) [reply UI/addEntry.html]: File names casing has been corrected

**Pull request merged**